commands/deploy_website.js

```diff
@@ -1,21 +1,17 @@
+const { readdir, hmac, feeconfirmation, setcontent, buildtxn, sendtxn, transfer, getindex } = require('../lib/functions')
 const fs = require('fs')
 const archethic = require('archethic')
-const originPrivateKey = "01009280BDB84B8F8AEDBA205FE3552689964A5626EE2C60AA10E3BF22A91A036009"
 const chalk = require('chalk')
 const mime = require('mime')
-const path = require('path')
 const crypto = require('crypto')
 const algo = 'sha256'
 const jsdom = require("jsdom")
 const { JSDOM } = jsdom
```

Comment:
With the right abstraction you should not have all those implementation details (like libraries, etc...) in this high level module.
The identification of the path and extraction is still in this command, which is the most important part (business logic) and most complex one. So this should be abstracted as well somewhere, maybe in its own module